go.mod

```diff
@@ -3,20 +3,21 @@ module github.com/open-cluster-management/governance-policy-status-sync
 go 1.16
 
 require (
-	github.com/onsi/ginkgo v1.14.1
-	github.com/onsi/gomega v1.10.1
-	github.com/open-cluster-management/addon-framework v0.0.0-20210415135036-d591b71789d6
-	github.com/open-cluster-management/governance-policy-propagator v0.0.0-20210330231547-932d5d73e0c3
-	github.com/operator-framework/operator-sdk v0.19.4
+	github.com/onsi/ginkgo v1.16.4
+	github.com/onsi/gomega v1.13.0
+	github.com/open-cluster-management/addon-framework v0.0.0-20210621074027-a81f712c10c2
```

Comment:
This should be `-io`? right?

Reply:
Seems like we're importing both? Because the file also has `open-cluster-management.io/addon-framework` ...

I'd like to defer fixing that to another PR (in either repo).